stripe/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -830,6 +790,52 @@ class CardInputWidget @JvmOverloads constructor(
         internal var dateRightTouchBufferLimit: Int = 0
         internal var cvcStartPosition: Int = 0
 
+        internal val cvcDestination: Int
+            @JvmSynthetic
+            get() {
+                return peekCardWidth + cardDateSeparation + dateWidth + dateCvcSeparation
+            }
+
+        @JvmSynthetic
+        internal fun getDateMargin(cardNumberIsViewed: Boolean): Int {
+            return if (cardNumberIsViewed) {
+                cardWidth + cardDateSeparation
+            } else {
+                peekCardWidth + cardDateSeparation
+            }
+        }
+
+        @JvmSynthetic
+        internal fun getCvcMargin(cardNumberIsViewed: Boolean): Int {
+            return if (cardNumberIsViewed) {
+                totalLengthInPixels
+            } else {
+                peekCardWidth + cardDateSeparation + dateWidth + dateCvcSeparation
+            }
+        }
+
+        @JvmSynthetic
+        internal fun updateSpacing(
+            isCardViewed: Boolean,
+            frameStart: Int,
+            frameWidth: Int
+        ) {
+            if (isCardViewed) {
+                cardDateSeparation = (frameWidth - cardWidth - dateWidth)
```

Comment:
out of curiosity, why do we need parens here?

Reply:
We don't, I clean this up in a future PR